tests/framework/extensions/workloads/pods/verify.go

```diff
@@ -0,0 +1,65 @@
+package pods
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	provv1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	client2 "github.com/rancher/rancher/pkg/client/generated/project/v3"
```

Comment:
A small nit; Could we call this alias something different, projectv3 or something?

Reply:
Sure